guides/common/modules/proc_installing-the-discovery-service.adoc

```diff
@@ -1,10 +1,70 @@
 [id="Installing_the_Discovery_Service_{context}"]
 = Installing the Discovery service
 
-Complete the following procedure to enable the Discovery service on {SmartProxyServer}.
+ifdef::satellite[]
```

Comment:
I'm a bit concerned about the length of this module and the repetition. Is it possible to use attributes and conditions so that steps are not duplicated? Otherwise you might have maintenance issues in the future.

Reply:
I think that this way is the most suitable. Further attributes and conditions would make it less readable.